loky/process_executor.py

```diff
@@ -969,25 +969,32 @@ def map(self, fn, *iterables, **kwargs):
     def shutdown(self, wait=True, kill_workers=False):
         mp.util.debug('shutting down executor %s' % self)
         self._flags.flag_as_shutting_down(kill_workers)
-        if self._queue_management_thread:
+        qmt = self._queue_management_thread
+        qmtw = self._queue_management_thread_wakeup
+        if qmt:
+            self._queue_management_thread = None
             # Wake up queue management thread
-            self._queue_management_thread_wakeup.wakeup()
+            if qmtw is not None:
+                qmtw.wakeup()
```

Comment:
This can fail with `OSError` if it has been closed concurrently so adding a `try..catch` is probably safer.